dataflux_pytorch/benchmark/llama-70b/load.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+import torch
+
+from dataflux_pytorch import dataflux_checkpoint
+
+start_time = time.time()
+ckpt = dataflux_checkpoint.DatafluxCheckpoint(project_name="<PROJECT_NAME>",
+                                              bucket_name="<BUCKET_NAME")
+CKPT_PATH = "llama2/llama2-70b-hf/"
+LOAD_COUNT = 10
+files = [
+    'pytorch_model-00001-of-00015.bin', 'pytorch_model-00002-of-00015.bin',
+    'pytorch_model-00003-of-00015.bin', 'pytorch_model-00004-of-00015.bin',
+    'pytorch_model-00005-of-00015.bin', 'pytorch_model-00006-of-00015.bin',
+    'pytorch_model-00007-of-00015.bin', 'pytorch_model-00008-of-00015.bin',
+    'pytorch_model-00009-of-00015.bin', 'pytorch_model-00010-of-00015.bin',
+    'pytorch_model-00011-of-00015.bin', 'pytorch_model-00012-of-00015.bin',
+    'pytorch_model-00013-of-00015.bin', 'pytorch_model-00014-of-00015.bin',
+    'pytorch_model-00015-of-00015.bin'
+]
+
+times = 0
+for i in range(LOAD_COUNT):
+    print(f"--------- RUN {i} ----------")
+    print("Performing read...")
+    for f in files:
+        with ckpt.reader(CKPT_PATH + f) as reader:
+            read_state_dict = torch.load(reader)
```

Comment:
If we are just benchmarking read code should we not have start_time just before the loop starts instead of including it at the top of the file (although I do understand it wont result in any major changes in terms of final result)

Reply:
Good catch. The bench was actually run that way to begin with, my formatting got messed up and it was put to the top when I was cleaning it up for github.